confgenerator/testdata/valid/linux/logging-complex_logs/input.yaml

```diff
@@ -40,11 +40,11 @@ logging:
       listen_host: 2.2.2.2
       listen_port: 2222
       transport_protocol: udp
-  exporters:
-    google:
-      type: google_cloud_logging
   service:
     pipelines:
+      default_pipeline:
```

Comment:
Are we now allowing users to disable the default pipeline like this?

Reply:
Resolved offline.

```
      default_pipeline:
        receivers: []
        exporters: []
```

is the expected syntax.